Analysis/Tasks/spectra.cxx

```diff
@@ -0,0 +1,49 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include <TH1F.h>
+
+#include <cmath>
+
+#include "Analysis/TrackSelection.h"
+#include "Analysis/TrackSelectionTables.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+//--------------------------------------------------------------------
+// Task generating pT spectrum of charged particles
+//--------------------------------------------------------------------
+struct SpectraAnalysisTask {
+  OutputObj<TH1F> pt{TH1F("pt", "pt", 100, 0., 50.)};
+
+  void init(o2::framework::InitContext &) {}
+
+  void process(aod::Collision const &collision,
+               soa::Join<aod::Tracks, aod::TrackFilterTable> const &tracks) {
+    for (auto &track : tracks) {
+      if (!track.isGlobalTrack())
+        continue;
+      pt->Fill(track.pt());
+    }
+  }
+};
+
```

Comment:
Once the filter support this, we should make a second example task here using a filter on the bit table